gui/src/bitcoind.rs

```diff
@@ -22,22 +22,22 @@ use std::os::windows::process::CommandExt;
 const CREATE_NO_WINDOW: u32 = 0x08000000;
 
 /// Current and previous managed bitcoind versions, in order of descending version.
-pub const VERSIONS: [&str; 4] = ["26.1", "26.0", "25.1", "25.0"];
+pub const VERSIONS: [&str; 5] = ["27.1", "26.1", "26.0", "25.1", "25.0"];
 
 /// Current managed bitcoind version for new installations.
 pub const VERSION: &str = VERSIONS[0];
 
 #[cfg(all(target_os = "macos", target_arch = "x86_64"))]
-pub const SHA256SUM: &str = "acb50edd20692a9d023de12da573b64ca0fd9b4e9a2b88d1251020a3022b0f27";
+pub const SHA256SUM: &str = "25445080e2616d7c621f48f51e7a4294698d92016ef27ab88608cbe824f93364";
```

Comment:
This should also be for the tar.gz file.

Reply:
sorry, done